docs/source/concepts/identity-and-package-management.rst

```diff
@@ -0,0 +1,170 @@
+.. Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Identity and Package Management
+###############################
+
+A DAML Ledger is a software system that enables parties to automate the management of their rights and obligations through smart contract code.
+The :ref:`DA Ledger Model <da-ledgers>` defines the permissible changes to the contracts (the integrity model), as well as the visibility that the different parties should have into such changes (the privacy model).
+This document addresses two additional topics:
+
+1. Management of parties' digital identifiers in a DAML Ledger.
+
+2. Distribution of smart contract code between the parties connected to the same DAML Ledger.
+
+The Ledger API provides both a :ref:`party management service <com.digitalasset.ledger.api.v1.PartyService>` and a :ref:`package service <com.digitalasset.ledger.api.v1.PackageService>`.
+The access to these services is usually more restricted compared to the other Ledger API services, as they are part of the administrative API.
+Any implementation of the services is guaranteed to accept inputs and provide outputs of the format specified by these services.
+However, the services' *behavior* -- the relationship between the inputs and outputs that the various parties observe -- is largely implementation dependent.
+The remainder of the document will presents both:
+
+#. the minimal behavioral guarantees for identity and package services across all ledger implementations, and
+
+#. guidelines to understand how the :ref:`ledger's topology <daml-ledger-topologies>` influences the unspecified part of the behavior.
+
+Identity Management
+*******************
+
+A DAML Ledger may freely define its own format of party and :ref:`participant node <participant-node-def>` identifiers, with some minor constraints on the identifiers' serialized form.
+For example, a ledger may use human-readable strings as identifiers, such as "Alice" or "Alice's Bank".
+A different ledger might use public keys as identifiers, or the keys' fingerprints.
+The applications should thus not rely on the format of the identifier -- even a software upgrade of a DAML Ledger may introduce a new format.
+
+By definition, identifiers identify parties, and are thus unique for a ledger.
+They do not, however, have to be unique across different ledgers.
+That is, two parties with identical identifiers in two different ledgers are not the same party.
```

Comment:
Is this true in a Canton world?
If I have two parties with the same identifier in two un-connected Canon domains, this would suggest they are different parties. However, as soon as these domains somehow get connected, even if through several domain-hops, they are the same? Or do you consider all Canton domains in the world to be part of one partitioned leder?